src/FileSystemUtilities.js

```diff
@@ -51,7 +51,7 @@ export default class FileSystemUtilities {
 
   @logger.logifyAsync()
   static rimraf(filePath, callback) {
-    rimraf(filePath, callback);
+    ChildProcessUtilities.spawn(join(__dirname, "../node_modules/.bin/rimraf"), [filePath], {}, callback);
```

Comment:
$(npm bin)${path.sep}rimraf ${filePath}? I'm not sure this'll work as written on Windows.